data/mherzog4.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Matt Herzog",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Engineer",
+  "avatar": "https://github.com/mherzog4.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mattherzog5",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://twitter.com/https://www.linkedin.com/in/mattherzog3/",
```

Comment:
Please fix this invalid URL! 🛠️

```suggestion
      "url": "https://www.linkedin.com/in/mattherzog3/",
```

Reply:
Sorry about that - was doing it from my phone.

Thanks for flagging.